simplegrid/cow.py

```diff
@@ -37,20 +38,21 @@ def to_observation(self, offset):
         return direction[0] + offset, direction[1] + offset
 
 
-class AbstractCow(abc.ABC):
+class AbstractCreature(abc.ABC):
```

Comment:
I'd suggest moving this to its own file, now that it is no longer, well, a cow

Reply:
Done